src/types.tsx

```diff
@@ -1,5 +1,8 @@
 export interface Campaign {
-  fallenHeroes: string[];
-  restOfAliveHeroes: string[];
-  allHeroes: string[];
+  allHeroes: Hero[];
 }
+
+type Hero = {
+  name: string;
+  alive: Boolean;
```

Comment:
```suggestion
  alive: boolean;
```
Explanation: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean